### PR TITLE
labels for oee

### PR DESCRIPTION
**Pull request by rssag**

### CODE REVIEW

File: content/oee/oee-administration-bundle/creating-line-profiles.md [hunk 1]
```diff
@@ -5,32 +5,32 @@ layout: redirect
 ---
 The following section walks you through the steps required to create a new line profile.
 
-### Step 1 - Create a new site
+### Step 1 - Create a new site {#step-1---create-a-new-site}
```
Comment (MWindel) [inline content/oee/oee-administration-bundle/creating-line-profiles.md:8]: @rssag How to deal with this case (several others in this file)?

File: content/oee/oee-faq-bundle/platform.md [hunk 2]
```diff
@@ -4,39 +4,39 @@ title: Cumulocity IoT
...
+### Where can I find the device and profile ID? {#where-can-i-find-the-device-and-profile-id}
 
 You can find the device ID and profile ID at the end of the URL shown in your browser when accessing the Machine Dashboard. The device ID comprises of the first seven numbers at the end of the URL. The remaining nine numbers are the profile ID.
 
 The example below shows an exemplary URL with the placement of both device ID and profile ID.
 
 `https://<<tenant>>/apps/oee/index.html#/machine_dashboard/<<device ID>>/<<profile ID>>
 `
-###  Why does a measurement/event/alarm in the OEE application have multiple timestamps?
+###  Why does a measurement/event/alarm in the OEE application have multiple timestamps? {#-why-does-a-measurementeventalarm-in-the-oee-application-have-multiple-timestamps}
```
Comment (MWindel) [inline content/oee/oee-faq-bundle/platform.md:25]:
```suggestion
### Why does a measurement/event/alarm in the OEE application have multiple timestamps? {#why-does-a-measurementeventalarm-in-the-oee-application-have-multiple-timestamps}
```

File: content/oee/oee-theory-bundle/calculation-logic-variables.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ title: Calculation logic & variables
 layout: redirect
 ---
 
-### Ideal Cycle Amount and Ideal Cycle Time (1), (2), (3)
+### Ideal Cycle Amount and Ideal Cycle Time (1), (2), (3) {#ideal-cycle-amount-and-ideal-cycle-time-1-2-3}
```
Comment (MWindel) [inline content/oee/oee-theory-bundle/calculation-logic-variables.md:7]: @rssag How should we deal with parentheses?

File: content/oee/oee-theory-bundle/calculation-pathways.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ In the following, details on each calculation pathway are provided.
 The tables at the top show which variables, starting from the input variables, are needed to calculate all other variables including the OEE. The different colors used in the images are used only for better understanding and clarity. Below the image you find the corresponding formulas. The formulas which are necessary to calculate the OEE are marked in italics in the table.
 {{< /c8y-admon-info >}}
 
-### Calculation pathway #1
+### Calculation pathway #1 {#calculation-pathway-1}
```
Comment (MWindel) [inline content/oee/oee-theory-bundle/calculation-pathways.md:17]: @rssag How should we deal with "#"?

File: content/oee/oee-theory-bundle/splitting.md [hunk 5]
```diff
@@ -6,21 +6,21 @@ layout: redirect
 
 The term "Splitting" describes the allocation of machine data to the OEE input parameters. This sections shows examples which explain the methodology used.
 
-### Input variable calculation - Example 1
+### Input variable calculation - Example 1 {#input-variable-calculation---example-1}
```
Comment (MWindel) [inline content/oee/oee-theory-bundle/splitting.md:9]: @rssag 3 slashes. See also below.

File: content/oee/oee-faq-bundle/calculation.md [hunk 6]
```diff
@@ -4,22 +4,22 @@ title: Calculation
 layout: redirect
 ---
 
-### How accurate are the calculations of the OEE application?
+### How accurate are the calculations of the OEE application? {#how-accurate-are-the-calculations-of-the-oee-application}
```
[the remark this answers is not in the document]
Comment (rssag) [reply content/oee/oee-faq-bundle/calculation.md:7]: the label is fine in this case

File: content/oee/oee-theory-bundle/calculation-logic-variables.md (hunk 3 above)
Comment (rssag) [reply content/oee/oee-theory-bundle/calculation-logic-variables.md:7]: fine in this case

File: content/oee/oee-theory-bundle/calculation-pathways.md (hunk 4 above)
Comment (rssag) [reply content/oee/oee-theory-bundle/calculation-pathways.md:17]: the label is fine in this case

File: content/oee/oee-theory-bundle/splitting.md (hunk 5 above)
Comment (rssag) [reply content/oee/oee-theory-bundle/splitting.md:9]: that's fine

File: content/oee/oee-administration-bundle/matching-data.md [hunk 7]
```diff
@@ -72,19 +72,19 @@ Of course, all three input parameters can also be derived directly from MEAs, or
...
 
-### Measurements
+### Measurements {#measurements}
 
 The following image provides an overview on the matching capabilities for measurements.
 
 ![Measurement](/images/oee/administration/matching-measurements.png)
 
-### Events & alarms
+### Events & alarms {#events--alarms}
```
[the remark this answers is not in the document]
Comment (rssag) [reply content/oee/oee-administration-bundle/matching-data.md:81]: fine in this case

**Pull request merged**